Duplicati/Library/Main/Database/LocalListDatabase.cs

```diff
@@ -367,13 +373,22 @@ public IEnumerable<IFileset> QuickSets
                                 @"SELECT DISTINCT ""ID"", ""IsFullBackup"" FROM ""Fileset"" ORDER BY ""Timestamp"" DESC ")
                         )
                         {
+                            // partial backup sets are only included until the first full backup is encountered
+                            var isFullBackupEncountered = false;
                             while (rd.Read())
                             {
                                 var id = rd.GetInt64(0);
-                                var isFullBackup = rd.GetInt32(1);
+                                var isFullBackup = rd.GetInt32(BackupType.FULL_BACKUP);

```

Comment:
To me, the name `isFullBackup` seems more fitting for a `bool` instead of an `int`.  Would it be clearer to define this as
```
var isFullBackup = rd.GetInt32(BackupType.FULL_BACKUP) == BackupType.FULL_BACKUP;
```

Reply:
Good catch on the "1". 

It could be a bool but I need the value when creating the Fileset().

Reply:
I agree that an "is" should be a bool. I'll change the variable from isFullBackup to backupType